tests/cli/build/plugin_tests.py

```diff
@@ -0,0 +1,259 @@
+        assert "android.permission.INTERNET" in output
+
+    @unittest.skipIf(Settings.HOST_OS != OSType.OSX, 'iOS tests can be executed only on macOS.')
+    def test_311_plugin_platforms_should_not_exist_in_tnsmodules_ios(self):
+        """
+        Test for issue https://github.com/NativeScript/nativescript-cli/issues/3932
+        """
+        Tns.platform_remove(app_name=self.app_name, platform=Platform.IOS)
+        Tns.plugin_add(plugin_name='nativescript-ui-listview', path=self.app_name, verify=False)
+        Folder.clean(os.path.join(self.app_name, 'node_modules'))
+        File.delete(os.path.join(self.app_name, 'package.json'))
+        copy = os.path.join('assets', 'issues', 'nativescript-cli-3932', 'nativescript-ui-listview')
+        paste = os.path.join(self.app_name, 'nativescript-ui-listview')
+        Folder.copy(copy, paste)
+        src = os.path.join('assets', 'issues', 'nativescript-cli-3932', 'package.json')
+        target = os.path.join(self.app_name)
+        File.copy(src=src, target=target)
+        Tns.platform_add_ios(app_name=self.app_name, framework_path=Settings.IOS.FRAMEWORK_PATH)
+        folder_path = os.path.join(self.app_name, 'nativescript-ui-listview')
+        Npm.install(option='--ignore-scripts', folder=folder_path)
+        Tns.build_ios(app_name=self.app_name)
+        app_path = os.path.join(self.app_name, Settings.Paths.PLATFORM_IOS)
+        assert not File.exists(os.path.join(app_path, 'nativescript-ui-listview', 'node_modules',
+                               'nativescript-ui-core', 'platforms'))
+
+    @unittest.skipIf(Settings.HOST_OS != OSType.OSX, 'iOS tests can be executed only on macOS.')
+    def test_320_CFBundleURLTypes_overridden_from_plugin_ios(self):
+        """
+        Test for issue https://github.com/NativeScript/nativescript-cli/issues/2936
+        """
+        Tns.platform_remove(app_name=self.app_name, platform=Platform.IOS)
+        plugin_path = os.path.join(Settings.TEST_RUN_HOME, 'assets', 'plugins', 'CFBundleURLName-Plugin.tgz')
+        Tns.plugin_add(plugin_path, path=self.app_name)
+        Tns.prepare_ios(app_name=self.app_name)
+        plist = File.read(os.path.join(self.app_name, Settings.Paths.PLATFORM_IOS, self.app_name,
+                                       self.app_name + "-Info.plist"))
+        assert "<key>NSAllowsArbitraryLoads</key>" in plist, \
+            "NSAppTransportSecurity from plugin is not found in final Info.plist"
+        assert "<string>bar</string>" in plist, "CFBundleURLTypes from plugin is not found in final Info.plist"
+
+    @unittest.skipIf(Settings.HOST_OS != OSType.OSX, 'iOS tests can be executed only on macOS.')
+    def test_401_plugin_add_invalid_plugin_ios(self):
+        Tns.platform_remove(app_name=self.app_name, platform=Platform.IOS)
+        Tns.platform_remove(app_name=self.app_name, platform=Platform.ANDROID)
+        result = Tns.plugin_add(plugin_name='wd', path=self.app_name, verify=False)
```

Comment:
We have the same in `test_400_plugin_add_invalid_plugin_android` why we need to duplicate it for iOS?